ckanserviceprovider/web.py

```diff
@@ -437,7 +437,8 @@ def job_status(job_id, show_job_key=False, ignore_auth=False):
     :statuscode 404: job id not found
     :statuscode 409: an error occurred
     '''
-    job_dict = db.get_job(job_id)
+    limit_logs = flask.request.args.get('limit_logs')
```

Comment:
Can you rename the API param to just `limit`? It's more consistent with others used elsewhere

Also please add the new param to the docstring